lib/ADS1219/native/mock_ADS1219.cpp

```diff
@@ -1,12 +1,18 @@
-#include "Arduino.h"
-
 #include <Wire.h>
 #include "ADS1219.h"
 
 ADS1219::ADS1219(int drdy, uint8_t addr, TwoWire *wire) {
   // do nothing.
 }
 
+ADS1219::ADS1219() {
+
+}
+
+void ADS1219::init(int drdy, uint8_t addr, TwoWire *wire) {
+  // do nothing.
+}
+
 long ADS1219::getData(uint8_t conf) {
   // do nothing.
```

Comment:
would be nice if there were an option to make this produce fake/random data, instead of just doing nothing. Would mean can still test things like connection to the dashboard (and could even become a tool itself for verifying the dashboard is graphing correctly w/o needing the actual PCB).